src/adapters/openai/completions.ts

```diff
@@ -58,10 +69,18 @@ The YAML parser that will be used to validate your output is shown below. Ensure
 
       parserCode,
 
-      `REFERENCE MANIFESTS:
+      `IMPORTANT CONTEXT MANIFESTS:
```

Comment:
How did you come to this conclusion that this change is needed?

Reply:
When I tried adding new plugin configurations, sometimes the manifests were ignored. Changing this to `IMPORTANT CONTEXT` caused the model to pick it up in most cases.